packages/components/src/toggle-group-control/toggle-group-control-option-base/component.tsx

```diff
@@ -141,10 +141,14 @@ function ToggleGroupControlOptionBase(
 					<Ariakit.Radio
 						disabled={ disabled }
 						onFocusVisible={ () => {
+							const selectedValueIsNullish =
```

Comment:
Naming nit: `nullish` often refers to `null` or `undefined`, so if someone isn't reading closely, they might be confused and expect this checks for `undefined` and not an empty string.

Reply:
I'd be fine with inlining it 👍

Reply:
Good point, I'll go with `selectedValueIsEmpty` here. (It started as an inline condition, but it was hard to parse/understand so I prefer it named.)